### DISCUSSION TITLE
PIjuice zero led

### DISCUSSION THREAD

**Discussion by xaja34** [PiSupply/PiJuice#968] [General]
Is there a way to turn off the PIJuice Zero Led ? I'm putting the PI zero with camera in a nesting box and I don't want to distract the birds. Thanks

**Comment by tvoverbeek** [comment]
In the PiJuice LED Settings (https://github.com/PiSupply/PiJuice/tree/master/Software#pijuice-hat-config-leds-menu) set all R,G,B values to zero.

**Reply by xaja34** [reply]
Thanks a lot.  Worked like a charm. I should have figured that out.

**Comment by tvoverbeek** [comment]
Good luck with your bird watching.
